src/Foundatio.Redis/Messaging/RedisMessageBus.cs

```diff
@@ -30,7 +30,7 @@ public class RedisMessageBus : MessageBusBase, IMessageBus {
                     return;
 
                 _logger.Trace("Subscribing to topic: {0}", _topic);
-                _subscriber.Subscribe(_topic, OnMessage);
+                _subscriber.Subscribe(_topic, OnMessage, CommandFlags.FireAndForget);
```

Comment:
This seems pretty iffy. I would want the messagebus to blow up if I did subscribe and it failed to subscribe.